### ISSUE TITLE
SEO Audit Failure: link.href.toLowerCase is not a function

### ISSUE THREAD

**Issue by cmoss3** [GoogleChrome/lighthouse#6481]
#### Provide the steps to reproduce
1. Run LH on https://www.browardcriminalteam.com/

#### What is the current behavior?
The SEO portion of the audit is not passing, nor is it providing explanation.

Under the SEO portion of the audit results I see "Error!" in the top right corner.
When I hover over the text an error message is revealed: "Audit Error: link.href.toLowerCase is not a function"
Here is a snapshot for context: https://www.screencast.com/t/XESTDL0iW

#### Environment Information
* Affected Channels: DevTools
* Lighthouse version: 3.0.3
* Operating System: Windows 10

**Comment by patrickhulce** [comment]
Thanks very much for reporting @cmoss3! It's the links inside SVG that are throwing this one :)

**Comment by cmoss3** [comment]
@patrickhulce Thanks for looking into this! I guess I was assuming that if an aria-label attribute was available on the link, then that would be sufficient for links with images/graphics inside. Either way though, it definitely shouldn't be causing the SEO audit to fail. Thanks again!

**Comment by patrickhulce** [comment]
Agreed, definitely wasn't a commentary on the way it's setup! Just explaining why LH is barfing :)

This is actually a great point I'm not sure if search will use the `aria-label` if there's no text available...

@rviscomi do you know if search has a public position on this?

**Comment by rviscomi** [comment]
No I don't believe aria-label is used as a fallback to missing inner text.
